modules/otel-config-instrumentation.adoc

```diff
@@ -98,9 +98,46 @@ spec:
 
 |===
 
+[id="otel-configuration-of-opentelemetry-sdk-variables_{context}"]
+== Configuration of the OpenTelemetry SDK variables
+
+You can use the `instrumentation.opentelemetry.io/inject-sdk` annotation in the OpenTelemetry Collector custom resource to instruct the {OTELName} Operator to inject some of the following OpenTelemetry SDK environment variables, depending on the `Instrumentation` CR, into your pod:
+
+* `OTEL_SERVICE_NAME`
+* `OTEL_TRACES_SAMPLER`
+* `OTEL_TRACES_SAMPLER_ARG`
+* `OTEL_PROPAGATORS`
+* `OTEL_RESOURCE_ATTRIBUTES`
+* `OTEL_EXPORTER_OTLP_ENDPOINT`
+* `OTEL_EXPORTER_OTLP_CERTIFICATE`
+* `OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE`
+* `OTEL_EXPORTER_OTLP_CLIENT_KEY`
+
+.Values for the `instrumentation.opentelemetry.io/inject-sdk` annotation
+[cols="a, a"]
+|===
+| Value | Description
+
+|`"true"`
+|Injects the `+Instrumentation+` resource with the default name from the current namespace.
+
+|`"false"`
+|Injects no `+Instrumentation+` resource.
```

Comment:
It means that this setting does not inject `+Instrumentation+`, right? 

Can it be revised something like that?